config/middlewares/authentication.js

```diff
@@ -23,4 +23,4 @@ module.exports = (req, res, next) => {
       })
       .catch(err => res.status(404).json(err));
   });
-};
\ No newline at end of file
+};
```

Comment:
Always add a new line at the end of files

Reply:
Okay Thanks, I will implement the feedback and raise another PR.